### PR TITLE
[IMP] l10n_din5008 add nl po files to all din5008 modules

### PR DESCRIPTION
**Pull request by jort-aardug**
Description of the issue/feature this PR addresses:
Add Dutch translation files for all l10n_din5008 modules
- l10n_din5008
- l10n_din5008_purchase
- l10n_din5008_repair
- l10n_din5008_sale
- l10n_din5008_stock

Current behavior before PR:
No Dutch translation files

Desired behavior after PR is merged:
Dutch translation files, available to be translated

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jort-aardug** [comment]
@mart-e , Pinged you yesterday but there were some issues regarding CLA so I closed that one. I was told I could ping you, could you take a look at this so we can translate these modules to dutch (through Transifex)?

**Comment by mart-e** [comment]
@jort-aardug hello :)
Thanks for the contribution but it seems these files are mostly untranslated. These localisation files are not on Transifex so the idea is to translate it here on Github directly.
You can use an external software like PoEdit to help you translate them.
After I can merge this PR

**Comment by jort-aardug** [comment]
> @jort-aardug hello :) Thanks for the contribution but it seems these files are mostly untranslated. These localisation files are not on Transifex so the idea is to translate it here on Github directly. You can use an external software like PoEdit to help you translate them. After I can merge this PR

Hi @mart-e , thanks for the reply! I discussed this previously with Yenthe, and he instructed me to add the PO files "empty" so we can then add them to Transifex to translate.
But if you want me to add translations here that's fine, I'll update the PR.

**Comment by mart-e** [comment]
Not sure what he means, the translations are always transifex -> github, never github -> transifex. And as these modules are not published on Transifex, it's best to do everything here

**Comment by jort-aardug** [comment]
Hi @mart-e , I've just added dutch translations to the po files. Could you have these reviewed? Thanks.

**Comment by mart-e** [comment]
@jco-odoo LGTM, is your team taking care of this kind of PR?

File: addons/l10n_din5008/i18n/nl.po [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+#. module: l10n_din5008
+#: model:ir.model.fields,field_description:l10n_din5008.field_base_document_layout__city
+msgid "City"
+msgstr "Plaats"
+
+#. module: l10n_din5008
+#: model:ir.model,name:l10n_din5008.model_base_document_layout
+msgid "Company Document Layout"
+msgstr "Bedrijf document lay-out"
```
Comment (jco-odoo) [inline addons/l10n_din5008/i18n/nl.po:100]: Documentlay-out bedrijf?

File: addons/l10n_din5008/i18n/nl.po [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+#: model:ir.model.fields,field_description:l10n_din5008.field_base_document_layout__company_registry
+msgid "Company ID"
+msgstr "Bedrijfs ID"
+
+#. module: l10n_din5008
+#. odoo-python
+#: code:addons/l10n_din5008/models/account_move.py:0
+#, python-format
+msgid "Credit Note"
+msgstr "Creditfactuur"
```
Comment (jco-odoo) [inline addons/l10n_din5008/i18n/nl.po:112]: Creditnota?

**Review by jco-odoo — COMMENTED** [review]
Not a translation specialist myself.

File: addons/l10n_din5008/i18n/nl.po (hunk 1 above)
Comment (jort-aardug) [reply addons/l10n_din5008/i18n/nl.po:100]: This, same with "creditfactuur", we're taken from existing translations in the account module. This one could be both but for the sake of consitency "Bedrijf document lay-out" I think is best. "Creditfactuur" should be correct as that's the most common translation dutch translations of "Credit Note" (within Odoo)

**Comment by jco-odoo** [comment]
@william-andre

**Comment by jort-aardug** [comment]
Hi, any update here or actions required from me? Thanks

**Comment by william-andre** [comment]
Sorry for the delay, I was on holiday
Would you mind just fixing your commit message to make the linter and CI happy? :smile:

https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html#commit-message-structure

**Comment by jort-aardug** [comment]
Hi @william-andre , no problem, holiday is always important :)
I believe it should be correct now, if not please let me know.

**Review by william-andre — APPROVED** [review]
It is good enough!

@robodoo r+

**Comment by william-andre** [comment]
Thank you for the contribution!

**Comment by william-andre** [comment]
My bad, I thought it would be a quick win

Here is the fix, tested for all the modules changed here
https://github.com/odoo/odoo/pull/133741

**Pull request closed without merging**